### ISSUE TITLE
wikidata identifier link in SRS stats

### ISSUE THREAD

**Issue by myrmoteras** [plazi/treatmentBank#54]
might it be possible to add the wikidata identifier for treatments to the SRS stats? 
https://tb.plazi.org/GgServer/srsStats

![image](https://user-images.githubusercontent.com/4609956/182580793-19c7dd37-bcbf-4911-ab2e-9417c8b01423.png)
 
because this is very helpful to get all at once
![image](https://user-images.githubusercontent.com/4609956/182580928-beb85ccd-e211-49b3-9a6d-26ee9a38036f.png)

**Comment by gsautter** [comment]
Sure, was the plan right from the start ... just wanted to wait for Andra to add more IDs, as right now this field has a non-empty value in merely a single treatment, and a more steady influx is yet to be set up.

Also, should we add the taxon concept ID only, or also the taxon name ID (maybe under "Taxonomic Data")?

**Comment by gsautter** [comment]
Anyway, added both IDs to the treatment stats now ... we might have to put all these external IDs into a dedicated table, as the main field group is getting quite full and hard to overview ...

**Comment by myrmoteras** [comment]
thanks -

**Comment by myrmoteras** [comment]
20230115

Hi Donat,
 
     I worked on the bot this weekend and added a few dozen treatments with its link to the papers and taxa to Wikidata.
 
Currently in Wikidata: https://w.wiki/6E8R
 
I have request permission to run this bot at scale. When this is approved (7 days to 3 month (or more)) we can do all.
 
Edits: https://www.wikidata.org/w/index.php?title=Special:Contributions/BiodiversityBot&target=BiodiversityBot&offset=&limit=500
 
Cheers, 

Andra

---------------

You might want to keep an eye on this development to fetch the wikidata id?!
I'm sure interested in this ... however, _fetching_ the IDs isn't the plan ... I created a dedicated microservice last summer (also as an architecture sanity check) when I created the one for the specimen keys coming in from the matching service, so Andra can readily send us the IDs - mailed him the credentials (API token) last summer, but sure can do so again once this starts to fly.

Cheers,
Guido
